CsvSplitter.js

```diff
@@ -10,7 +10,7 @@ CsvSplitter.prototype = {
     split: function(filepath, maxEntries, ouputDirectory) {
         this._csvReader = new CsvReader(filepath);
 
-        this._amountOfFiles = ~~(this._csvReader.length() / maxEntries);
+        this._amountOfFiles = ~~((this._csvReader.length()-1) / maxEntries)+1;
```

Comment:
exclude header from line count `(this._csvReader.length()-1)`
include remainder lines to last file `+1`